src/Amazon.Extensions.CognitoAuthentication/CognitoUserAuthentication.cs

```diff
@@ -178,13 +178,40 @@ public async Task<AuthFlowResponse> RespondToSmsMfaAuthAsync(RespondToSmsMfaRequ
         /// if one exists</returns>
         public async Task<AuthFlowResponse> RespondToNewPasswordRequiredAsync(RespondToNewPasswordRequiredRequest newPasswordRequest)
         {
+            var result = await RespondToNewPasswordRequiredAsync(newPasswordRequest, null).ConfigureAwait(false);
```

Comment:
no need to await here, right?

Reply:
The original method has the async keyword. Can that cause an issue if the async keyword is dropped?

Reply:
You can remove both the `async` and `await` keywords from this method. `async` doesn't effect the signature of the method, it is just instructions for the compiler when building your method. You can safely remove the `async` keyword from the method as long as your method continues to return a `Task`.

Reply:
Thanks for chiming in. fixed it.